### PR TITLE
devide WebDriverFactory, create DriverContainer

### PR DESCRIPTION
**Pull request by TarasTsyhanko**

### CODE REVIEW

File: src/main/java/bo/SearchActions.java [hunk 1]
```diff
@@ -18,17 +18,17 @@
 
     @Step("open website : {url}")
     public void openWebSite() {
```
Comment (mtsyhanko) [inline src/main/java/bo/SearchActions.java]: you have the same method in CartActions class

Comment (mtsyhanko) [reply src/main/java/bo/SearchActions.java]: Better extract this method to some other class
Create some class 
for example CommonActions

File: src/main/java/factory/DriverFactory.java [hunk 2]
```diff
@@ -25,22 +25,21 @@
...
     public static void runWithZeroImplicitly(Supplier<WebElement> supplier) {
-        setWait(getDiver(), 0);
+        setWait(DriverContainer.getDiver(), 0);
         supplier.get();
-        setWait(getDiver(), implicitlyWait);
+        setWait(DriverContainer.getDiver(), implicitlyWait);
     }
 
-    private static void initDriver() {
+    public static WebDriver initDriver() {
```
Comment (mtsyhanko) [inline src/main/java/factory/DriverFactory.java]: rename initDriver to createDriver

**Review by mtsyhanko — APPROVED** [review]

File: src/main/java/factory/DriverFactory.java (hunk 2 above)
Comment (TarasTsyhanko) [reply src/main/java/factory/DriverFactory.java]: fixed

File: src/main/java/bo/SearchActions.java (hunk 1 above)
Comment (TarasTsyhanko) [reply src/main/java/bo/SearchActions.java]: fixed

**Pull request merged**